### ISSUE TITLE
Error when Building the export templates from linux to android (thirdparty/vulkan/loader/loader.c: fatal error: 'loader_cmake_config.h' file not found)

### ISSUE THREAD

**Issue by Federico-Ciuffardi** [godotengine/godot#36921]
**Godot version:**
https://github.com/Federico-Ciuffardi/godot/commit/24c94a877ee83d6a4b02a984aae4eb9b69522ad7

**OS/device including version:**
Linux 5.5.7-1-MANJARO

**Issue description:**
I tried to build the export templates using the following [guide](http://docs.godotengine.org/en/stable/development/compiling/compiling_for_android.html#building-the-export-templates). When executing `scons platform=android target=release android_arch=armv7` this error happens:
```
[ 45%] Compiling ==> thirdparty/vulkan/loader/loader.c
thirdparty/vulkan/loader/loader.c:81:10: fatal error: 'loader_cmake_config.h' file not found
#include "loader_cmake_config.h"
         ^~~~~~~~~~~~~~~~~~~~~~~
```
  
**Steps to reproduce:**
* I tried to build the export templates for android using the following: https://github.com/Federico-Ciuffardi/godot/commit/24c94a877ee83d6a4b02a984aae4eb9b69522ad7
* And followed the steps of this guide http://docs.godotengine.org/en/stable/development/compiling/compiling_for_android.html#building-the-export-templates

**Comment by akien-mga** [comment]
The Android platform has not been ported to support Vulkan yet, so this is expected.

**Comment by Federico-Ciuffardi** [comment]
Is there any way to compile despite this?

**Comment by akien-mga** [comment]
Yes, you can merge #36919 locally.

**Comment by akien-mga** [comment]
Fixed by #36919.
